src/app/records/record-list/record-list.component.scss

```diff
@@ -40,7 +40,7 @@
       width: 100px
     ),
     cctDate: (
-      width: 110px
+      width: 130px
```

Comment:
Proper @Health-Education-England/frontend / UI Design question but what does this do for the target device width?

Reply:
This change increases the width of the "Curriculum end date column". As the column title has been changed longer now, the width needed to be longer so it won't break to another line. I believe the whole table width would be increased by 20px, but with my local testing, it is still fitting on the same page without needing to scroll on my screen. For the mobile setting, the table would need to be scroll originally anyway.